jobs/frontend/senior-frontend-developer.md

```diff
@@ -14,17 +14,38 @@ We're looking for a pro-active senior-level frontend developer, who not only sol
 ## We expect that you
 
 - Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important.
+- Write readable and maintainable code.
+- Care about user experience.
+- Have knowledge of best practices in development and in UI/UX design.
 - Can design a whole system of application.
-- Knows at least one library/framework for building UI at advanced level (React, Vue, Angular, etc.).
-- Have experience working with different kind of teams and projects.
```

Comment:
I decided to remove it because I think it's important for us to find a developer with very strong skills and such developer might not have experience working with different kind of teams (though probably have experience working with different kind of projects).